app/controllers/concerns/spina/current_spina_account.rb

```diff
@@ -0,0 +1,28 @@
+module Spina
+  module CurrentSpinaAccount
+    extend ActiveSupport::Concern
+    
+    included do
+      before_action :current_spina_account
+      helper_method :current_spina_account
+      helper_method :current_account
+    end
+  
+    private
+
+      unless Spina.disable_deprecated_methods
```

Comment:
Shouldn't this be `Spina.config.disable_deprecated_methods`?

Reply:
Yeah, I will change that.